examples/timestamping/backend/src/lib.rs

```diff
@@ -77,3 +78,29 @@ impl Service for TimestampingService {
         TimestampingApi.wire(builder);
     }
 }
+
+impl Configure for TimestampingService {
+    type Params = Config;
+
+    fn verify_config(
```

Comment:
Since `initialize` method and `Configure` interface for `TimestampingService` share the same configuration structure, you can add `impl Config` with verification and applying code to avoid code duplication.